Adding postman scripts (#18)
aweber/public-api-examples: postman/README.md

# Postman code examples

This directory contains examples for use with Postman

> Script | Link |

Reviewer: Is it valid markdown to drop the opening `|`?

Author: Added the opening |

--- | --- |
Create and Schedule Broadcast|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8c408d4c102f2152f5c5)
Find Subscribers Across Lists|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/6f8ba7670cc386207673)

> Get Broadcast Details|[[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/20540bbd207861516f95)

Reviewer 2: ```suggestion
Get Broadcast Details|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/20540bbd207861516f95)
```

> Get Broadcast Open Click|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/31f187b357a84c3a2ca8)

Reviewer 2: ![screen shot 2018-11-26 at 10 56 37 am](https://user-images.githubusercontent.com/3092064/49025582-00f9d300-f16a-11e8-87df-187f0c9e1bc8.png)
this imports as `Get Broadcast Email Clicks` don't know how that title is different.

Reviewer 2: nvm these are getting updated last, after all changes

Get Broadcast Stats|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/007cb5e575b093b53444)
Get Clicks|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/3ef21f19bcf8bc8d7504)
Get Integrations|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/8f521e5bd78c83b9e8e8)
Get Tags|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
Get Webforms Detail|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2e2c3ee4ff07d60146db)
Manage Custom Fields|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/46ea48d4f90e6595d421)
Manage Subscriber|
Move Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/37ffdf13b6588329b6da)
Please follow this tutorial to import this [Postman Collection](https://www.getpostman.com/docs/collections).

> ## Keys used in APIs that need to be created by the user:

Reviewer 2: include a link on how to make these keys or a few steps.

> ### For all requests

Reviewer: There should be defaults set for this now

> ### For all requests

Author: I've added base_url to each collection

- consumer_key
- consumer_secret
- access_token

> - secret_token

Reviewer 2: `secret_token`? ive never seen it called that before, you mean `token_secret` or `access_token_secret`?

Reviewer: Yeah this should be token_secret

> - access_secret

Reviewer: All the collections use `token_secret` not `access_secret`

Author: Updated

> 

Reviewer: To create and schedule a broadcast, all of these environment variables are needed:
* body_html
* body_text
* click_tracking_enabled
* exclude_lists
* facebook_integration
* include_lists
* is_archived
* notify_on_send
* subject
* twitter_integration

If you don't specify them, postman sends the template placeholder as the value:
```
body_html=%7B%7Bbody_html%7D%7D&body_text=test&click_tracking_enabled=%7B%7Bclick_tracking_enabled%7D%7D&exclude_lists=%7B%7Bexclude_lists%7D%7D&facebook_integration=%7B%7Bfacebook_integration%7D%7D&include_lists=%7B%7Binclude_lists%7D%7D&is_archived=%7B%7Bis_archived%7D%7D&notify_on_send=%7B%7Bnotify_on_send%7D%7D&subject=A%20broadcast&twitter_integration=%7B%7Btwitter_integration%7D%7D
```

I recommend hard coding values for all of these since I think it's unreasonable to require this many inputs to run the collection.

> ## Keys used in APIs that need to be created by the user:

Reviewer 2: ```suggestion
## Environment variables used in APIs that need to be created by the user:
```

> - secret_token

Reviewer 2: ```suggestion
- token_secret
```

| Get Integrations|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/0566aa365b24d75822e9)|
| Get Tags|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/223a97960ea532ae59cd)|
| Get Webforms Detail|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a88306f31ebe6c8e1949)|
| Manage Custom Fields|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/966f607e6a5b3b78751c)|
| Manage Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/23943d73ae00a6371eb5)|
| Move Subscriber|[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/2ced13b02af75197126a)|

Please follow this tutorial to import this [Postman Collection](https://www.getpostman.com/docs/collections)

## Environment variables that need to be created in Postman:

### For all requests

- consumer_key
- consumer_secret
- access_token
- token_secret

> ### To search for a subscriber by email across all lists

Reviewer: This is also used to add a new subscriber in Manage Subscriber

Author: Added
